README.md

```diff
@@ -118,85 +128,99 @@ Since the docker-postgis repository is an extension of the official Docker Postg
+
+Read more in the [docker-postgres README page](https://github.com/docker-library/docs/blob/master/postgres/README.md)
 
 Warning: **the Docker specific variables will only have an effect if you start the container with a data directory that is empty;** any pre-existing database will be left untouched on container startup.
 
 It's important to note that the environment variables for the Docker image are different from those of the [libpq — C Library](https://www.postgresql.org/docs/current/libpq-envars.html)  (`PGDATABASE`,`PGUSER`,`PGPASSWORD` )
 
-## Troubleshooting tips:
+## Troubleshooting tips
 
 Troubleshooting can often be challenging. It's important to know that the docker-postgis repository is an extension of the official Docker PostgreSQL repository. Therefore, if you encounter any issues, it's worth testing whether the problem can be reproduced with the [official PostgreSQL Docker images](https://hub.docker.com/_/postgres). If so, it's recommended to search for solutions based on this. The following websites are suggested:
 
-* Upstream docker postgres repo: https://github.com/docker-library/postgres
-  * search for the open or closed issues !
-* Docker Community Forums: https://forums.docker.com
-* Docker Community Slack: https://dockr.ly/slack
-* Stack Overflow: https://stackoverflow.com/questions/tagged/docker+postgresql
+* Upstream [docker postgres repository](https://github.com/docker-library/postgres)
+  * search for the open or closed issues!
+* [Docker Community Forums](https://forums.docker.com)
+* [Docker Community Slack](https://dockr.ly/slack)
+* [Stack Overflow](https://stackoverflow.com/questions/tagged/docker+postgresql)
 
 If your problem is Postgis related:
 
-* Stack Overflow : docker + postgis https://stackoverflow.com/questions/tagged/docker+postgis
-* Postgis issue tracker: https://trac.osgeo.org/postgis/report
+* Stack Overflow: [docker + postgis](https://stackoverflow.com/questions/tagged/docker+postgis)
+* [Postgis issue tracker](https://trac.osgeo.org/postgis/report)
 
 And if you don't have a postgres docker experience - read this blog post:
 
-* https://www.docker.com/blog/how-to-use-the-postgres-docker-official-image/
-
+* [docker documentation](https://www.docker.com/blog/how-to-use-the-postgres-docker-official-image/)
 
 ## Security
 
 It's crucial to be aware that in a cloud environment, with default settings, these images are vulnerable, and there's a high risk of cryptominer infection if the ports are left open. ( [Read More](https://github.com/docker-library/postgres/issues/770#issuecomment-704460980) )
+
 * Note that ports which are not bound to the host (i.e., `-p 5432:5432` instead of `-p 127.0.0.1:5432:5432`) will be accessible from the outside. This also applies if you configured UFW to block this specific port, as Docker manages its own iptables rules. ( [Read More](https://docs.docker.com/network/iptables/) )
 
-#### Recomendations:
+### Recomendations
```

Comment:
Typo fix opportunity:  I believe "Recomendations" should be spelled "Recommendations"